sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/PartitionKind.java

```diff
@@ -10,7 +10,13 @@
     /**
      * The Partition of a item is calculated based on the hash value of the PartitionKey.
      */
-    HASH("Hash");
+    HASH("Hash"),
+
+    RANGE("Range"),
```

Comment:
is this needed right now or it is just for future proofing the enum?

Reply:
Range support needs SDK changes for sure. 
Lets please add it only when its covered E2E

Reply:
Future proofing.